modules/apps/collaboration/wiki/wiki-editor-configuration/src/test/java/com/liferay/wiki/editor/configuration/internal/WikiAttachmentImageHTMLEditorConfigContributorTest.java

```diff
@@ -165,6 +170,11 @@ public void testItemSelectorURLWhenAllowBrowseAndValidWikiPage()
 		wikiAttachmentImageHTMLEditorConfigContributor.setItemSelector(
 			_itemSelector);
 
+		suppress(
```

Comment:
It doesn't seem to be right, because we are modifiying the behaviour of the logic we are testing.

It seems to be a dependency error (can you try to upgrade the kernel version in the test bundle and recheck??)